testing/jormungandr-testing-utils/src/wallet/account.rs

```diff
@@ -92,6 +92,10 @@ impl Wallet {
         ))
     }
 
+    pub fn internal_counters_increasing(&self) -> &SpendingCounterIncreasing {
```

Comment:
@dkijania did something different in #3735, so I think it may be better to do the same.